components/Nav/Nav.module.css

```diff
@@ -41,8 +41,8 @@
 }
 
 .logoLink {
-  height: calc(var(--navBarHeight) - 15px);
-  margin: 7.5px 15px;
+  height: calc(var(--navBarHeight) - 25px);
```

Comment:
the logo isn't vertically center aligned. it's a little closer to the top of the nav bar than the bottom

Reply:
changed from 25px to 21 and it is centered now. thank you for spotting it.